src/index.js

```diff
@@ -0,0 +1,25 @@
+const { exec } = require('child_process')
+
+module.exports = function (pluginConfig, config, cb) {
+  exec('git tag', (err, stdout, stderr) => {
+    if (err) return cb(err)
+
+    const versionRegex = /^v((?:[0-9]+\.){2}[0-9]+)$/
+
+    const tags = stdout
+                  .trim()
+                  .split('\n')
+                  .map((t) => t.trim())
+                  .filter(t => versionRegex.test(t))
```

Comment:
as for the order, It's not, I'll fix that.

As for the t thing, you're totally right. I'll fix that as well.